pepper-apis/src/main/java/org/broadinstitute/ddp/cache/LanguageStore.java

```diff
@@ -14,31 +17,26 @@
 
     public static final String DEFAULT_LANG_CODE = "en";
 
-    private static final Map<String, LanguageDto> languages = new HashMap<>();
-
-    public static LanguageDto get(String isoCode) {
-        return languages.get(isoCode);
-    }
+    private static Map<String, LanguageDto> languages = Collections.emptyMap();
 
-    public static LanguageDto getDefault() {
-        return languages.get(DEFAULT_LANG_CODE);
+    public static synchronized void init(Handle handle) {
+        Map<String, LanguageDto> map = new HashMap<>();
+        handle.attach(JdbiLanguageCode.class)
+                .findAll()
+                .forEach(lang -> map.put(lang.getIsoCode(), lang));
+        languages = Map.copyOf(map);
```

Comment:
The `copyOf` call isn't necessary here- assigning the `HashMap` instance we create in the first line should be sufficient.

Reply:
True, but it spells out that we're making an immutable map and I believe there's some optimizations under-the-hood too.

Reply:
Just brushed up on those docs- that's _nice_. I take my comment back- this is a great use of `copyOf`!